SignalAnalysis/localization/strings.hi.resx

```diff
@@ -0,0 +1,512 @@
+    <value>संभावना</value>
+  </data>
+  <data name="strPlotFractalDistributionTitle" xml:space="preserve">
+    <value>भग्न आयाम वितरण</value>
+  </data>
+  <data name="strPlotFractalTitle1" xml:space="preserve">
+    <value>भग्न आयाम</value>
+  </data>
+  <data name="strPlotFractalTitle2" xml:space="preserve">
+    <value>(संचयी)</value>
+  </data>
+  <data name="strPlotFractalXLabel" xml:space="preserve">
+    <value>समय (सेकंड)</value>
+  </data>
+  <data name="strPlotFractalYLabel" xml:space="preserve">
+    <value>आयाम (एच)</value>
+  </data>
+  <data name="strPlotOriginalTitle" xml:space="preserve">
+    <value>इनपुट संकेत</value>
+  </data>
+  <data name="strPlotOriginalXLabel" xml:space="preserve">
+    <value>समय (सेकंड)</value>
+  </data>
+  <data name="strPlotOriginalYLabel" xml:space="preserve">
+    <value>आयाम</value>
+  </data>
+  <data name="strPlotWindowTitle" xml:space="preserve">
+    <value>{0} खिड़की</value>
+  </data>
+  <data name="strPlotWindowXLabel" xml:space="preserve">
+    <value>समय (सेकंड)</value>
+  </data>
+  <data name="strPlotWindowYLabel" xml:space="preserve">
+    <value>आयाम</value>
+  </data>
+  <data name="strRadCurrentCulture" xml:space="preserve">
+    <value>वर्तमान संस्कृति स्वरूपण</value>
+  </data>
+  <data name="strRadInvariantCulture" xml:space="preserve">
+    <value>अपरिवर्तनीय संस्कृति स्वरूपण</value>
+  </data>
+  <data name="strRadPoints" xml:space="preserve">
+    <value>डेटा अंक</value>
+  </data>
+  <data name="strRadSeconds" xml:space="preserve">
+    <value>सेकंड</value>
+  </data>
+  <data name="strRadTime" xml:space="preserve">
+    <value>तिथि और समय</value>
+  </data>
+  <data name="strRadUserCulture" xml:space="preserve">
+    <value>संस्कृति का चयन करें</value>
+  </data>
+  <data name="strReadDataError" xml:space="preserve">
+    <value>फ़ाइल से डेटा पढ़ने में असमर्थ।</value>
+{0}</value>
+  </data>
+  <data name="strReadDataErrorCulture" xml:space="preserve">
+    <value>संस्कृति पहचानकर्ता स्ट्रिंग नाम मान्य नहीं है।
+{0}</value>
+  </data>
+  <data name="strReadDataErrorCultureTitle" xml:space="preserve">
+    <value>संस्कृति नाम त्रुटि</value>
+  </data>
+  <data name="strReadDataErrorNumber" xml:space="preserve">
+    <value>अमान्य अंकीय मान: {0}</value>
+  </data>
+  <data name="strReadDataErrorNumberTitle" xml:space="preserve">
+    <value>डेटा पार्स करने में त्रुटि</value>
+  </data>
+  <data name="strReadDataErrorTitle" xml:space="preserve">
+    <value>गलती</value>
+  </data>
+  <data name="strReadNotimplementedError" xml:space="preserve">
+    <value>किसी {0} फ़ाइल से डेटा पढ़ने में असमर्थ।</value>
+  </data>
+  <data name="strReadNotimplementedErrorTitle" xml:space="preserve">
+    <value>लागू नहीं किया गया</value>
+  </data>
+  <data name="strSaveDlgFilter" xml:space="preserve">
+    <value>Text file (*.txt)|*.txt|SignalAnalysis file (*.sig)|*.sig|Binary file (*.bin)|*.bin|Results file (*.results)|*.results|All files (*.*)|*.*</value>
```

Comment:
Is this OK in Hindi?

Reply:
Those are file types and extensions. In my opinion It's better that they remain in English.

Reply:
All right.